unit-tests/unit-tests-hdr.cpp

```diff
@@ -309,276 +309,312 @@ TEST_CASE("HDR Config while Streaming", "[hdr][live][using_pipeline]")
 TEST_CASE("HDR Running - restart hdr at restream", "[hdr][live][using_pipeline]")
 {
     rs2::context ctx;
-    if (make_context(SECTION_FROM_TEST_NAME, &ctx))
+    if (make_context(SECTION_FROM_TEST_NAME, &ctx, "2.39.0"))
     {
         rs2::device_list devices_list = ctx.query_devices();
         size_t device_count = devices_list.size();
         REQUIRE(device_count > 0);
 
         rs2::depth_sensor depth_sensor = restart_first_device_and_return_depth_sensor(ctx, devices_list);
 
-        rs2::config cfg;
-        cfg.enable_stream(RS2_STREAM_DEPTH);
-        rs2::pipeline pipe;
-        pipe.start(cfg);
+        if (depth_sensor && depth_sensor.supports(RS2_OPTION_SEQUENCE_ID) &&
+            depth_sensor.supports(RS2_OPTION_HDR_ENABLED))
+        {
```

Comment:
It seem that RS2_OPTION_SEQUENCE_ID is redundant, can you use only the latter ?

Reply:
You mean checking only if option HDR_ENABLED is supported, and not the SEQUENCE_ID option?

Reply:
Done